api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/AvailabilityHandler.java

```diff
@@ -113,6 +114,10 @@ public void createAvailabilityMetric(
             @ApiParam(required = true) MetricDefinition metricDefinition,
             @Context UriInfo uriInfo
     ) {
+        if(metricDefinition.getType() != null && metricDefinition.getType() != MetricType.AVAILABILITY) {
```

Comment:
That would change the current API, null should be allowed value here (it's already defined in the URL).